src/rustdoc-internals.md

```diff
@@ -213,3 +213,13 @@ $ python3 -m http.server -d build/[YOUR ARCH]/doc
 
 Now you can browse your documentation just like you would if it was hosted
 on the internet. For example, the url for `std` will be `/std/".
```

Comment:
I know you didn't add this, but could you fix this?

```suggestion
on the internet. For example, the url for `std` will be `/std/`.
```

Reply:
I think this can be fixed in a follow-up.